app/(dashboard)/mailboxes/[mailbox_slug]/common-issues/page.tsx

```
@@ -0,0 +1,378 @@
+
+  const bulkCloseAllMutation = api.mailbox.issueGroups.bulkCloseAll.useMutation({
+    onSuccess: (result) => {
+      toast.success(`Closed ${result.updatedCount} conversations`);
+      refetch();
+    },
+    onError: (error) => {
+      toast.error(error.message);
+    },
+  });
+
+  const handleBulkCloseAll = (groupId: number, cleanTitle: string, openCount: number) => {
```

ESLint

[error] 'cleanTitle' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)